config/100-watcher-serviceaccount.yaml

```diff
@@ -23,6 +23,10 @@ kind: ClusterRole
 metadata:
   name: tekton-results-watcher
 rules:
+  # Watcher needs to be able to create new and update existing results.
+  - apiGroups: ["results.tekton.dev"]
```

Comment:
Interesting, so it seems like the resources don't have to map to actual K8s CRD resources 🤯

Reply:
Yup! I was really please this Just Worked™️